### PR TITLE
README update

### PR DESCRIPTION
**Pull request by celinecamacho**
Add information about RxJava, MVVM and Scarlet.

### CODE REVIEW

File: fe2-android/README.md [hunk 1]
```diff
@@ -13,11 +13,16 @@ This repository is being refactored and some classes and interfaces marked with
...
-
+* The [R class in Android](https://stackoverflow.com/questions/4953077/what-is-the-class-r-in-android) is an auto-generated class containing the resource IDs all the resources of res/directory
+* The application follows the MVVM pattern, and uses this [guide to app architecture](https://developer.android.com/jetpack/guide)
+* The [ViewModel](https://developer.android.com/topic/libraries/architecture/viewmodel) classes implement the UI logic, retrieve and manage the data that is needed
+* The WebSockets are implemented using the [Scarlet](https://github.com/Tinder/Scarlet) library, one of its usage on this application is in the [Injection](https://github.com/dedis/student_21_pop/blob/master/fe2-android/app/src/prod/java/com/github/dedis/student20_pop/Injection.java) class
+* The application uses [RxJava](https://github.com/ReactiveX/RxJava) for asynchronous and event based programs
+* The the [Room](https://developer.android.com/reference/androidx/room/package-summary) library is used to define the application's database, entities and Data Access Objects
```
Comment (pierluca) [inline fe2-android/README.md:21]: The the ? :)

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -169,6 +169,7 @@ jobs:
         go test -race -v ./...
 
   fe2-android:
+    name: Fe2-Android
```
Comment (pierluca) [inline .github/workflows/ci.yaml:172]: thank you, sorry, I forgot that.

**Review by pierluca — COMMENTED** [review]
Minor issue found in the README :) 

Wherever it feels relevant, feel free to ask reviews from the students now :) 
It's a way to ease them into the codebase.

File: fe2-android/README.md (hunk 1 above)
Comment (celinecamacho) [reply fe2-android/README.md]: Oups

**Review by gnarula — APPROVED** [review]

**Pull request merged**